src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/model/community/Conviction.kt

```diff
@@ -0,0 +1,45 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.model.community
+
+import java.time.LocalDate
+import java.time.LocalDateTime
+
+data class Conviction(
+  val convictionId: Long,
+  val index: String,
+  val active: Boolean,
+  val inBreach: Boolean,
+  val failureToComplyCount: Long,
+  val breachEnd: LocalDate?,
+  val awaitingPsr: Boolean,
+  val convictionDate: LocalDate,
+  val referralDate: LocalDate,
+  val offences: List<Offence>?
+)
+
+data class Offence(
+  val offenceId: String,
+  val mainOffence: Boolean,
+  val detail: OffenceDetail,
+  val offenceDate: LocalDateTime,
+  val offenceCount: Long,
+  val tics: Long,
```

Comment:
I think we'll need to mark `tics` as optional as I don't see it in responses I've obtained...

Reply:
Ah cheers for this!